### ISSUE TITLE
Needs better documentation

### ISSUE THREAD

**Issue by stormhillmedia** [liximomo/vscode-sftp#367]
## Expected Behavior
explain how to use the extension beyond where to find config script and "enjoy"

## Actual Behavior

Usage says:

1)    Ctrl+Shift+P on Windows/Linux or Cmd+Shift+P on Mac open command palette, run SFTP: config command.
 2) Enjoy.

**Comment by zigojacko** [comment]
Agreed, taking Visual Studio Code for a test drive for the first time, opted on 'sftp' extension for handling local/remote file handling/syncing and the documentation is pretty much non-existent so stumbling around in the dark trying to figure out how the hell to use it and find where everything is... :/

**Comment by liximomo** [comment]
Need help.

**Comment by dooferorg** [comment]
The examples help some, but I did find that mine and my coworker's configurations seemed a bit different than the default. The default configuration does not contain a 'name' which seems to be suggested, also the port number is an integer and shouldn't be quoted as a string.

Further to that the uploadOnSave needs to be specifically stated, it won't automatically do it (something that confused me at first), and then there's the setting of a private key file which I also had to look up to see how to specify such things ( privateKeyPath ) - I use the exported output from puttygen to 'openssh [private] key' in the conversions menu in puttygen .
